doc/userguide/firewall/firewall-example.rst

```diff
@@ -0,0 +1,93 @@
+Firewall Ruleset Examples
```

Comment:
I think some very sipmle examples would be useful as well.
- allow all HTTP, tls, ssh, dns
- drop all

Or the better common case
- allow everything outgoing stateful
- only allow incoming ssh, HTTP, https